client/src/helpers/ManageBookings/formatBookingDate.ts

```diff
@@ -0,0 +1,6 @@
+export default function formatBookingDate(date: Date): string {
+  const day = date.getDate();
+  const month = new Intl.DateTimeFormat('en-US', { month: 'long' }).format(date);
+  const year = date.getFullYear();
+  return `${day} ${month} ${year}, 8-9`;
```

Comment:
Is there a reason we hardcoded `8-9` here? Ideally, we should avoid hardcoding things in FE/BE tickets in order not to indroduce overhead for integration. More on this idea below as well 👍🏼

Reply:
No, this is a placeholder, and bad practice I imagine - I wanted to get the feature pushed.  I forgot that I would be needing to display time, so I hard-coded a placeholder.